src/components/introduction.stories.mdx

```diff
@@ -49,26 +50,28 @@ const Root = () => (
   </ThemeProvider>
 );
 ```
+
 3. Join a meeting
 
 To join a meeting, you need to do the following sub steps:
 
-* Fetch the meeting and attendee data from Chime
+- Fetch the meeting and attendee data from Chime
 
-* Intialize `MeetingSessionConfiguration`
+- Intialize `MeetingSessionConfiguration`
```

Comment:
nit: Intialize -> Initialize (sorry not your change but can we fix it :)  ?)

Reply:
Updated.